client/rest-high-level/src/main/java/org/elasticsearch/client/IndicesClient.java

```diff
@@ -155,6 +159,96 @@ public Cancellable createAsync(CreateIndexRequest createIndexRequest,
             CreateIndexResponse::fromXContent, listener, emptySet());
     }
 
+    /**
+     * Creates a data stream using the Create Data Stream API.
+     * See <a href="https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-data-streams.html">
+     * Data Streams API on elastic.co</a>
+     * @param createDataStreamRequest the request
+     * @param options the request options (e.g. headers), use {@link RequestOptions#DEFAULT} if nothing needs to be customized
+     * @return the response
+     * @throws IOException in case there is a problem sending the request or parsing back the response
+     */
+    public AcknowledgedResponse createDataStream(CreateDataStreamRequest createDataStreamRequest,
+                                                RequestOptions options) throws IOException {
```

Comment:
Super minor, but the indentation is off for a few of these methods in this file

Reply:
Fixed